dashboard/lib/tasks/seed_in_test.rake

```diff
@@ -1,4 +1,7 @@
 Rake::Task['db:test:prepare'].enhance do
+  if ENV['UTF8']
+    system('mysql -uroot -e "ALTER DATABASE dashboard_test CHARACTER SET utf8 COLLATE utf8_unicode_ci;"')
```

Comment:
is there a way we can execute this SQL through the dashboard-configured SQL connection, rather than going through the CLI? Hardcoding in an execution of `mysql -uroot` doesn't seem super desirable

Reply:
I don't know the answer, but I'm open to specific suggestions!

Reply:
I'm thinking something like `ActiveRecord::Base.connection.execute`, with the caveat that it probably needs to go after the `ActiveRecord::Base.establish_connection(:test)` on the next line.

Actually, we could probably even do something like

```suggestion
    ActiveRecord::Base.connection.execute("ALTER DATABASE #{ActiveRecord::Base.connection.current_database} CHARACTER SET utf8 COLLATE utf8_unicode_ci;")
```

to avoid hardcoding the name of the database while we're at it.

Reply:
that seems to work, thanks!